src/build.py

```diff
@@ -694,6 +693,23 @@ def Which(name):
     return ''
 
 
+def CopyLLVMTools(build_dir, install_subdir=''):
+  # The following isn't useful for now, and takes up space.
+  Remove(os.path.join(INSTALL_DIR, install_subdir, 'bin', 'clang-check'))
+  # The following are useful, LLVM_INSTALL_TOOLCHAIN_ONLY did away with them.
+  extra_bins = ['FileCheck', 'lli', 'llc', 'llvm-as', 'llvm-dis', 'llvm-link',
+                'llvm-nm', 'opt']
+  extra_libs = ['libLLVM*.so']
```

Comment:
I think you can remove this line now that I fixed it upstream to be installed with LLVM_INSTALL_TOOLCHAIN_ONLY